TTS/elevenlabs.py

```diff
@@ -0,0 +1,55 @@
+import random
+
+from elevenlabs import generate, save
+
+from utils import settings
+
+voices = [
+    "Adam",
+    "Antoni",
+    "Arnold",
+    "Bella",
+    "Domi",
+    "Elli",
+    "Josh",
+    "Rachel",
+    "Sam",
+]
+
+class elevenlabs:
+    def __init__(self):
+        self.max_chars = 5000
+        self.voices = voices
```

Comment:
are you sure it is 5000 chars on the website it is only 2500 per request.

Reply:
Can you link this? I don't believe there is a limit with the api. It may be better to lower this amount though - I was just matching the pyttsx structure

Reply:
I've changed to 2500 though